conf/config.json

```diff
@@ -13,11 +13,13 @@
         "erc1155_gateway": "0xa5Df8530766A85936EE3E139dECE3bF081c83146",
         "dai_gateway": "0x0000000000000000000000000000000000000000",
         "usdc_gateway": "0x0000000000000000000000000000000000000000",
-        "lido_gateway": "0x0000000000000000000000000000000000000000"
+        "lido_gateway": "0x0000000000000000000000000000000000000000",
```

Comment:
Should we add the wrapped token gateway as well to track eth deposit?

Reply:
or does it use the following gateway?
```
    /// @notice The address of ERC20 gateway used to bridge WETH.
    address public immutable gateway;
```

Reply:
It use L1StandardERC20Gateway.

Reply:
got it.